app/views/manifestation/browse.html.haml

```diff
@@ -1,32 +1,28 @@
 -# Browse works
 #content.container-fluid
   .row.list-all-texts
-    .col-12.col-lg-4
-      .by-card-v02.desktop-only#browse_intro{style:"display:#{ @filters.empty? ? 'show' : 'none'}"}
+    .col-sm-12.col-md-4
+      .by-card-v02#browse_intro{ style: "display:#{@filters.empty? ? 'block' : 'none'}" }
         .work-area
           .work-content
             .headline-1-v02= t(:welcome_to_works_browsing)
             %p!= t(:works_browsing_intro_html)
           .work-side-color
             .work-genre.prose-side-icon
-      - if @filters.empty?
-        .by-card-v02.text-sorting#sort_filter_panel{style: 'display:none'}
-          .by-card-header-v02
-            .headline-2-v02= t(:sort_by)
-          .by-card-content-v02#filters_panel
-            .temporary
-      - else
-        .by-card-v02.text-sorting#sort_filter_panel{style: 'display:show'}
-          = render partial: 'browse_filters'
-    .col-12.col-lg-8
+      .by-card-v02.text-sorting#sort_filter_panel{ style: 'display:none' }
```

Comment:
shouldn't this be conditional upon filters.empty? too?

Reply:
Thanks, @abartov a good catch!
But we cannot decide if element should be shown or not on server side, as we need to know if this is a mobile view or desktop.
So I've added JS code to handle this.

Also added couple of other adjustements:
- browse_intro snippet should not be shown on mobile at all (to copy old behaviour)
- added scrolling to the top of the list when mobile filter is applied.

And modified authors browse page accordingly.